### PR TITLE
On drag do not snap sticker to center

### PR DESCRIPTION
**Pull request by marcianosr**

### CODE REVIEW

File: src/css/header.css [hunk 1]
```diff
@@ -11,7 +11,7 @@ header {
 	justify-content: space-between;
 }
 
-a {
+header a {
```
Comment (RoelN) [inline src/css/header.css]: Hmm, this keeps popping up in PRs?

**Comment by RoelN** [comment]
@marcianosr Could you take a look at the conflict? I'll merge after that!

File: src/js/main.js [hunk 2]
```diff
@@ -168,11 +168,11 @@ let maxStickableY;
 const sticker = {
 	current: false,
 	updateSticker: function() {
-		this.x = mouse.x + document.documentElement.scrollLeft;
+		this.x = mouse.x - sticker.offsetX;
 		this.y = Math.max(
 			minStickableY,
 			Math.min(
-				mouse.y + document.documentElement.scrollTop,
```
Comment (marcianosr) [inline src/js/main.js]: This line was still there that caused a bug.

File: src/js/main.js [hunk 3]
```diff
@@ -202,12 +202,18 @@ const sticker = {
...
+		const offsetLeft = parseInt(e.target.style.getPropertyValue("--x"), 10);
+		const offsetTop = parseInt(e.target.style.getPropertyValue("--y"), 10);
+		sticker.offsetX = mouse.x - offsetLeft;
+		sticker.offsetY = mouse.y - offsetTop;
 		// Move clicked sticker
 		sticker.current = e.target;
 		sticker.current.classList.add("dragging");
 	} else {
 		// Create new sticker
+		sticker.offsetX = 0;
```
Comment (marcianosr) [inline src/js/main.js]: Also, when generating a sticker, there is no offset set, which will result in `NaN` when the updateSticker function runs.

**Comment by RoelN** [comment]
Nice work, thanks @marcianosr!

**Pull request merged**